easybuild/easyblocks/c/clang.py

```diff
@@ -84,16 +94,26 @@ def __init__(self, *args, **kwargs):
         self.llvm_obj_dir_stage3 = None
         self.make_parallel_opts = ""
 
-        unknown_targets = [target for target in self.cfg['build_targets'] if target not in CLANG_TARGETS]
+        build_targets = self.cfg['build_targets']
+        if build_targets is None:
+            arch = get_cpu_architecture()
+            default_target = TARGETS_MAP.get(arch, None)
+            if default_target:
+                self.cfg['build_targets'] = build_targets = default_target
```

Comment:
This should be using `[default_target]` rather than `default_target`, since a list is expected later to define `DLLVM_TARGETS_TO_BUILD`:

```python
';'..join(self.cfg['build_targets'])
```